app/services/scrapers/high_priority.rb

```diff
@@ -18,7 +18,9 @@ def perform
       bar = ProgressBar.new(cases.count)
       puts "#{cases.count} are high priority for update for #{county.name} county"
       cases.each do |case_number|
-        court_case = ::CourtCase.find_by!(county_id: @county.id, case_number: case_number)
+        court_case = ::CourtCase.find_by(county_id: @county.id, case_number: case_number)
+
+        next unless court_case.nil? == false
         next unless court_case.enqueued == false
```

Comment:
Instead of the double negative, what would you think about:

```suggestion
        next if court_case.enqueued
```